docs/style-guide/lists/index.md

```diff
@@ -0,0 +1,240 @@
+  - title: Hyperledger
+    meta: John Owens
+    link: https://hyperledger.org/
+---
+
+# Component Usage
+
+## Standard Lists
+
+Standard lists can be called anywhere in markdown by using
+
+```
+- Lorem ipsum
+- Si dolor et amet
+- nunc habitasse portitor demi
+```
+
+Ensure that the list is always preceeded with one of the following, taking care to ensure that your markup is semantic:
+
+```
+Your heading text {.list-title}
+# Your heading text {.list-title}
+## Your heading text {.list-title}
+### Your heading text {.list-title}
+#### Your heading text {.list-title}
+##### Your heading text {.list-title}
+###### Your heading text  {.list-title}
+```
+
+{.mb-4}
+
+--- {.mb-4}
+
+### Examples{.mt-8}
+
+#### Unordered List {.list-title .mt-4}
+
+- Lorem ipsum si dolor et amet
+- Nunc habitasse et portitor demi
+
+Markdown:
+
+```
+#### List Example One {.list-title}
+- Lorem ipsum si dolor et amet
+- Nunc habitasse et portitor demi
+```
+
+#### Links with descriptions {.list-title .mt-8}
+
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+  {.list-link-description}
+
+Markdown:
+
+```
+#### Links with descriptions {.list-title}
+
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+{.list-link-description}
+```
+
+#### Links with descriptions without bullets {.list-title .mt-8}
+
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+  {.no-bullets .list-link-description}
+
+Markdown:
+
+```
+#### Links with descriptions {.list-title}
+
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+{.no-bullets .list-link-description}
+```
+
+#### Ordered List {.list-title .mt-8}
+
+1. Lorem ipsum si dolor et amet
+2. Nunc habitasse et portitor demi
+
+Markdown:
+
+```
+#### Ordered List Example {.list-title}
+1. Lorem ipsum si dolor et amet
+2. Nunc habitasse et portitor demi
+```
+
+## Rich Lists
+
+There are multiple versions of lists to be generated. To use them, create a frontmatter array of objects, with the following properties:
+
+- `title: required` The title text
+- `subtitle: optional` If used, this will render a second line below the title.
+- `meta: optional` If used, this will render a new column to the right (desktop). Use for things such as dates.
+- `link: required` This will be the `href`.
+- `internal: optional (Boolean)` Use internal: true when passing internal links.
```

Comment:
Again, let's just use that `isExternal` helper function in the `list-card` component vs. requiring the user to  specify whether it's internal/external.